corehq/apps/hqwebapp/static/hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.js

```diff
@@ -138,6 +138,18 @@ $.widget("ui.multiselect", {
 
 		$.widget.prototype.destroy.apply(this, arguments);
 	},
+	refresh: function() {
+		this.container.remove();
+		this._create();
+	},
+	addSelected: function(value, text) {
+		this.element.append(
+			$("<option/>").prop('value', value)
+			              .prop('selected', 'selected')
+			              .text(text)
+		);
+		this.refresh();
+	},
```

Comment:
select2 would definitely be a better option here. jquery ui stuff isn't super great for what we need it, and we use select2 a bunch elsewhere. https://select2.github.io/